soc/riscv/riscv-ite/it8xxx2/Kconfig.defconfig.series

```diff
@@ -26,6 +26,10 @@ config UART_NS16550_WA_ISR_REENABLE_INTERRUPT
 	default y
 	depends on UART_NS16550
 
+config UART_ITE_IT8XXX2
+	default y
+	depends on UART_NS16550
```

Comment:
This change also moves the UART config declare from the board level to the soc level.